addons/mail/static/src/components/thread_view_top_bar/thread_view_top_bar.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.ThreadViewTopbar" owl="1">
+        <div class="o_ThreadViewTopbar">
+            <span class="o_ThreadViewTopbar_member">
+                <t t-esc="thread.name"/>
+                (<t t-esc="thread.members.length"/> members)
```

Comment:
avoid building string this way, this should come from a %s string to be translatable correctly